pos_report_session_summary/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# copyright Odoo SA
+# copyright OCA
+# Licence : LGPL-3
+
+{
+    'name': 'POS report Session Summary',
+    'version': '10.0.1.0.0',
+    'category': 'Point Of Sale',
+    'summary': 'Backport of the report from 9.0',
```

Comment:
I think it's not a good summary for a module. The summary is better a short resume of the module.